test/selenium/testlib_avocado/seleniumlib.py

```diff
@@ -46,6 +48,8 @@
 actualpath = "."
 IDENTITY_FILE = "identity"
 
+LOGGER.setLevel(logging.WARNING)
```

Comment:
With this you set the default selenium warning to be WARNING? (A bit confused because the commit msg says suspend default selenium logging.

Reply:
Yep, because WARNING should be probably enough, to see WARNINGS from selenium.
but yep, it is little bit confusing, that I'm saying suspend, but still WARN and above will be shown